### ISSUE TITLE
features.js should not .detach() the DOM

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#623]
```
12:
window.caniusecallback = function(data) {
  var dom = $('.support').detach();
...

This makes the new design disappear the bottom 1/2 of the page.
```

Original issue reported on code.google.com by `ericbide...@html5rocks.com` on 3 Oct 2011 at 3:48

**Comment by GoogleCodeExporter** [comment]
```
fixed in r201bca7fc54e01727b2baa81de8596d787cd4b1d
```

Original comment by `paulir...@google.com` on 5 Oct 2011 at 10:33
- Changed state: **Fixed**
